src/clusterfuzz/_internal/fuzzing/corpus_manager.py

```diff
@@ -243,6 +250,20 @@ def get_gcs_url(self, suffix=''):
 
     return url
 
+  def get_zipcorpus_gcs_dir_url(self):
+    """Build zipcorpus GCS URL for gsutil.
+    Returns:
+      A string giving the GCS URL.
+    """
+    url = storage.get_cloud_storage_file_path(self.bucket_name,
+                                              f'zc{self.bucket_path}')
```

Comment:
Rather than prepend 'zc', can we instead use a more descriptive name/path, e.g. "/zipped/" (and put that in a constant in the implementation here)?